PowerEditor/src/WinControls/Window.h

```diff
@@ -64,8 +64,20 @@ class Window
 			::UpdateWindow(_hSelf);
 	}
 
+	virtual void redrawDlgItem(const int nIDDlgItem, bool forceUpdate = false) const

```

Comment:
What does it do exactly this function?
And why do we need it?

Reply:
@donho 
currently using `redraw()` will redraw all items in window which is unnecessary and can cause flickering.
Try switching fast options in Preference->Editing->Current Line Indicator.

`MapWindowPoints` is used to get correct coordinates of child rect for redrawing.
https://stackoverflow.com/questions/18034975/how-do-i-find-position-of-a-win32-control-window-relative-to-its-parent-window

Reply:
@ozone10 
In this case, this method should be in `class StaticDialog` instead of `class Window`.